### DISCUSSION TITLE
Three suggestions.

### DISCUSSION THREAD

**Discussion by ghost** [vgmtrans/vgmtrans#382] [Music formats]
I was wondering if you'd accept the possibility of converting Nintendo 64 USF files. It's not listed, but maybe it would be more useful(?) to have that as a conversion choice.

Also, is it possible for WAV exporting to be an option for conversion? Like, exporting the samples from the converted files into a folder? I have OpenMPT and I use wav samples for the musical instruments, maybe that could work too.

Sorry, it's actually two. Pretty misleading.

Thank you. Hope you enjoyed my begging.

**Comment by sykhro** [comment]
> I was wondering if you'd accept the possibility of converting Nintendo 64 USF files. It's not listed, but maybe it would be more useful(?) to have that as a conversion choice.

Yes, pull requests are welcome. At the moment the project is mostly in maintainance mode since all of us maintainers are busy with other things and contribute features occasionally.

> Also, is it possible for WAV exporting to be an option for conversion? Like, exporting the samples from the converted files into a folder? I have OpenMPT and I use wav samples for the musical instruments, maybe that could work too.

If you right click on a sample collection, you can save all of its samples as WAV. We do plan on improving UX for this so that you can export all the wavs for all the open files with one click
<img width="341" alt="image" src="https://github.com/vgmtrans/vgmtrans/assets/8491400/60c4a40e-2577-4e70-ba96-1e493a52f305">
